src/ExpandablePicker/ExpandablePicker.js

```diff
@@ -346,6 +371,15 @@ module.exports = kind(
 	},
 
 	/**
+	* Update header accessibilityLabel whenever currentValueText changed.
+	*
+	* @private
+	*/
+	currentValueTextChanged: function () {
+		this.set('accessibilityText', this.multiSelectCurrentAccesibilityLabel());
```

Comment:
Please check accessibility option so that we can add this line only for accessibility option ON